Application/EdFi.Ods.Api/Security/Authorization/ViewNameHelper.cs

```diff
@@ -24,6 +24,11 @@ public static class ViewNameHelper
         /// <returns>The authorization view name (without the schema).</returns>
         public static string GetAuthorizationViewName(string endpoint1, string endpoint2, string authorizationPathModifier)
         {
+            if (endpoint1.Equals("StudentUSI") && endpoint2.Equals("EducationOrganizationId"))
+            {
+                return $"{endpoint1}To{endpoint2}{authorizationPathModifier}";
+            }
+
```

Comment:
Spin this entire static class of as a LegacyViewNameHelper, and write a replacement that follows the current conventions for the new views.  Mark anything legacy with the ticket number that will remove it (ODS-5040 in this case)  Adding this check risks some unexpected side effects, and doesn't handle the view name patterns consistently.